bwdata.py

```diff
@@ -0,0 +1,805 @@
+
+    def get_world(self, name=None, startDate=None, **kwargs):
+        """
+        Retrieves world overview (mentions map) data.
+
+        Args:
+            name:           You must pass in a query / group name (string).
+            startDate:      You must pass in a start date (string).
+            kwargs:         All other filters are optional and can be found in filters.py.
+        
+        Returns:
+            A dictionary representation of mapped mentions on a globe data
+        """
+        params = self._fill_params(name, startDate, kwargs)
+        return self.project.get(endpoint="data/volume/queries/countries", params=params)["results"]["values"]
+
+    def get_keyinsights(self, name=None, startDate=None, **kwargs):
+        """
+        Retrieves key insights component data.
+
+        Args:
+            name:           You must pass in a query / group name (string).
+            startDate:      You must pass in a start date (string).
+            kwargs:         All other filters are optional and can be found in filters.py.
+        
+        Returns:
+            A dictionary representation of the component key insights
+        """
+        key_insights = {"total_mentions":self.get_keyinsights_mention_count(name, startDate),
+        "unique_authors":self.get_keyinsights_author_count(name, startDate),
+        "topic_trends":self.get_keyinsights_topics(name, startDate),
+        "rising_news":self.get_keyinsights_news(name, startDate)}
+        return key_insights
+
+
+    def get_keyinsights_mention_count(self, name=None, startDate=None, **kwargs):
+        """
+        Retrieves total mentions count data from key insights component.
+
+        Args:
+            name:           You must pass in a query / group name (string).
+            startDate:      You must pass in a start date (string).
+            kwargs:         All other filters are optional and can be found in filters.py.
+        
+        Returns:
+            An integer that represents the total number of mentions
+        """
+        params = self._fill_params(name, startDate, kwargs)
+        return self.project.get(endpoint="data/mentions/count", params=params)["mentionsCount"]
+
+    def get_keyinsights_author_count(self, name=None, startDate=None, **kwargs):
+        """
+        Retrieves total unique authors count data from key insights component.
+
+        Args:
+            name:           You must pass in a query / group name (string).
+            startDate:      You must pass in a start date (string).
+            kwargs:         All other filters are optional and can be found in filters.py.
+        
+        Returns:
+            An integer that represents the total number of unique authors
+        """
+        params = self._fill_params(name, startDate, kwargs)
+        return self.project.get(endpoint="data/authors/months/queries", params=params)["results"][0]["values"][0]["value"] 
```

Comment:
What is all the other crap in the returned JSON?  For fuck's sake, Brandwatch...